lib/nylas/http_client.rb

```diff
@@ -111,11 +117,11 @@ def default_headers
     rescue JSON::ParserError
       response
     end
+    inform_on :parse_response, level: :debug, also_log: { result: true }
```

Comment:
Curious, what does `inform_on` do?

Reply:
inform_on logs the entire output of the `parse_response` if you have your NYLAS_LOG_LEVEL set to `debug` or greater verbosity. 

Very useful for diagnosis and exploration.